templates/index.html

```diff
@@ -1,9 +1,55 @@
 {% extends "base.html" %}
 
-{% block title %}Flasky{% endblock %}
+{% block title %}RuneScraper{% endblock %}
+
+{% block head %}
+    {{ super() }}
+    <link href='{{ url_for('static', filename='css/index.scss') }}' rel='stylesheet' type='text/css'>
+{% endblock %}
 
 {% block page_content %}
-<div class="page-header">
-    <h1>Hello World!</h1>
-</div>
+    <div class="row">
```

Comment:
is this wrapped in a "container" or "container-fluid" somewhere? Will cause headaches later if not.

Reply:
never mind - carousel